CalculatorTest.py

```diff
@@ -20,6 +20,9 @@ def test_run_operation_divide(self):
         with self.assertRaises(ZeroDivisionError):
             Calculator.run_operation(9, 0, '/')
 
+    def test_run_operation_modulo(self):
+        self.assertEqual(7, Calculator.run_operation(7, 10, '%'))
```

Comment:
This is a good test case. Maybe adding a test to make sure numbers that should give an error do give the right error.

Reply:
I completely agree. A good example, I believe, would also be how long the inputs from the user can be.

Reply:
yes, also test negative numbers